### PR TITLE
[RFR] Simplify makefile using environment variables

### PR DESCRIPTION
**Pull request by djhi**
Follow #135 

No more `deploy-[environment]' targets, just specify `NODE_ENV` to target the correct environment

### CODE REVIEW

**Comment by Kmaschta** [comment]
Can you rebase, please?

**Comment by djhi** [comment]
done :)

File: makefile [hunk 1]
```diff
@@ -16,11 +18,17 @@ help:
 
 # Initialization ===============================================================
 copy-conf: ## Initialize the configuration files by copying the *''-dist" versions (does not override existing config)
-	-cp -n ./config/development-dist.js ./config/development.js
+	-cp -n ./config/${NODE_ENV}-dist.js ./config/${NODE_ENV}.js
+ifeq ($(NODE_ENV), development)
```
Comment (Kmaschta) [inline makefile]: Are you sure about this? Don't you need backslashes here?

Comment (djhi) [inline makefile]: I'm sure it worked locally when I did:
```sh
NODE_ENV=test make install
```

Comment (djhi) [inline makefile]: and if it didn't work, travis won't pass because it'll miss the babel/webpack/selenium packages

**Comment by Kmaschta** [comment]
Please update the `doc/DEPLOY.md` file :)

**Pull request merged**